packages/mockotlpserver/lib/cli.js

```diff
@@ -118,6 +118,11 @@ const OPTIONS = [
         type: 'bool',
         help: 'Start a web server to inspect traces with some charts.',
     },
+    {
+        names: ['tunnel', 't'],
+        type: 'string',
+        help: `Set this option to a URL to send all requests to another server (only HTTP supported for now). This won't stop the processing of OTLP data.`,
```

Comment:
```suggestion
        help: `Tunnel all incoming requests to the given server. Only supported for the HTTP OTLP server (port 4318). Received OTLP data will still be printed per the '-o' option.`,
```

Suggested alternative, because "This won't stop the processing of OTLP data." confused me for a while.